packages/amplify-provider-awscloudformation/lib/system-config-manager.js

```diff
@@ -86,7 +87,10 @@ async function getProfiledAwsConfig(context, profileName, isRoleSourceProfile) {
   if (httpProxy) {
     awsConfig = {
       ...awsConfig,
```

Comment:
The linter seems to fail here.

Reply:
@willkara here's the lint failures:

$ eslint .

/home/circleci/repo/packages/amplify-provider-awscloudformation/src/aws-utils/aws-pinpoint.js
  46:5   error  'awsConfig' is not defined  no-undef
  47:10  error  'awsConfig' is not defined  no-undef

/home/circleci/repo/packages/amplify-provider-awscloudformation/src/aws-utils/aws.js
  15:5   error  'awsConfig' is not defined  no-undef
  16:10  error  'awsConfig' is not defined  no-undef

To fix them, from the root of your amplify cli repo - you can run run `yarn lint-fix` or `npm run lint-fix`

Reply:
Gotcha, I'll try that on my Linux machine as Windows was giving me issues for some reason.

Reply:
Everytime I try running it I get an error:
```

willkara@NORMANDY:~/amplify-cli$ eslint .

Oops! Something went wrong! :(

ESLint: 6.1.0.

ESLint couldn't find a configuration file. To set up a configuration file for this project, please run:

    eslint --init

ESLint looked for configuration files in /home/willkara/amplify-cli and its ancestors. If it found none, it then looked in your home directory.

If you think you already have a configuration file or if you need more help, please stop by the ESLint chat room: https://gitter.im/eslint/eslint
```

This is from a fresh clone and install.

```
willkara@NORMANDY:~/amplify-cli$ yarn lint-fix
yarn run v1.13.0
$ lerna run lint-fix
lerna notice cli v3.10.8
........
.........
$ eslint . --fix
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run lint-fix stderr:

Oops! Something went wrong! :(

ESLint: 6.1.0.

ESLint couldn't find the config "airbnb-base" to extend from. Please check that the name of the config is correct.

The config "airbnb-base" was referenced from the config file in "/home/willkara/amplify-cli/packages/amplify-category-api/.eslintrc.json".

If you still have problems, please stop by https://gitter.im/eslint/eslint to chat with the team.

error Command failed with exit code 2.
```

Reply:
So I had to init eslint but now it's giving me TONS of errors. Is there not a way to point to the packages eslint config?

Reply:
Still trying but every-time I run the linter, it's given me tons of other line-ending and other errors, even though I didn't change anything else.

@UnleashedMind / @kaustavghosh06  any ideas?

Reply:
@UnleashedMind / @kaustavghosh06 , hey guys any ideas on this? I've tried multiple linters/runs of it but it's giving me the same errors. I can upload a file of them if you'd like.